docs/reference/deploy/config/master-config-reference.rst

```diff
@@ -1595,3 +1595,11 @@ requests and responses.
 =================
 
 The path to the IdP's certificate, used to validate assertions.
+
+********************
+ ``reserved_ports``
+********************
+
+Determined makes use of certain ports for inter-process communication, however this may cause
```

Comment:
Shouldn't there be an example showing how to do this in the `master.yaml`?

Reply:
Added an example, and some bare bones information on port usage.